### ISSUE TITLE
Some problems with the max iters

### ISSUE THREAD

**Issue by powermano** [endernewton/tf-faster-rcnn#279]
I want to know why choosing max_iters=70000 for pascal_voc 2007 dataset? 
Will it decrease the 07 metric results when increasing the max_iters?
Recently i do not train the pascal_voc 2007 dataset from scratch, i first use part of the dataset  to initializing the network ,and then fine-tuning the network incrementally.
More  specifically, i first random choose 1000 images from the all dataset and train the network 70000 iters with the imagenet models.  Then i use some algorithms to select  another 1000 images from the rest of dataset and fine-tune the network with lower learning rate. But surprisingly , the fine-tuning results get 58.0 map and  training the network with 2000 images from scratch achieves 59.3.

I do not using any more tricks except for lowing the learning rate.  I want to know how to fine-tune the object detection network . 
Anybody can supply more fine-tuning tricks.

Thanks in advance.
